ui/txs/noves/useDescribeTxs.tsx

```diff
@@ -0,0 +1,89 @@
+import type { UseQueryResult } from '@tanstack/react-query';
+import _ from 'lodash';
+
+import type { NovesDescribeTxsResponse } from 'types/api/noves';
+import type { Transaction } from 'types/api/transaction';
+
+import type { ResourceError } from 'lib/api/resources';
+
+export interface DescribeTxs {
+  txHash: string;
+  type?: string;
+  description?: string;
```

Comment:
Maybe it is better to move these properties under `data` fields, so the structure will be similar to `QueryResult` type.

```
data: { type: string; description: string } | null;
isLoading: boolean;
```